app/controllers/friendships_controller.rb

```diff
@@ -1,12 +1,12 @@
 class FriendshipsController < ApplicationController
   def new
-    @friendship = Friendship.new
+    @friendship = Friendship.build
   end
 
   def create
     @friendship = current_user.friendships.new
     @friendship.user_id = current_user.id
-    @friendship.friend_id = params[:format]
+    @friendship.friend_id = friendship_params
```

Comment:
As indicated by the last reviewer, always use `build` instead of `new`

Between those two lines
```rb
@friendship.user_id = current_user.id
@friendship.friend_id = params[:format]
```
one is not needed as you instantiated the friendship directly with `current_user`